main_test.go

```diff
@@ -218,14 +231,56 @@ func TestRunInitActionCmd(t *testing.T) {
 			wantErr: true,
 		},
 		{
-			name: "invalid token prompt",
+			name:   "invalid token prompt",
+			client: client,
 			cmd: &InitActionCmd{
 				DirPath: "migrations",
 				Driver:  "mysql",
 			},
 			prompt:  "invalid token\n",
 			wantErr: true,
 		},
+		{
+			name: "file exists",
+			client: &githubClient{
```

Comment:
maybe add helper func: `createGHClient(repoSvc: Repositories)`.
and init the client using: `createGHClient(&mockService{})` or  `createGHClient(&mockService{getContentError: &github.ErrorResponse{Message: "Not Found"}})`